app/mailers/user_mailer.rb

```diff
@@ -115,7 +115,11 @@ def locked_time
   end
 
   def unlock_time
-    (@user.locked_at + 1.hour).to_fs(:govuk_date)
+    t = (@user.locked_at + 1.hour)
```

Comment:
I know it wasn't you who added it in the first place, but it would be better to use `User.unlock_in` here instead of `1.hour` - I believe the former is provided from the Devise config which I think what this `1.hour` refers to here.

Reply:
That's great, thank you for spotting it.